change paperspace with gradient (#18)
Paperspace/Docs: experiments/containers-public-and-private.md

  
Once you have downloaded your JSON key file, rename it to "keyfile.json".

> Use the contents of the keyfile as the password value in the gradient jobs create method:

Reviewer: ```suggestion
Use the contents of the keyfile as the password value in the gradient experiments method:
```

```text

> gradient jobs create ... --container "gcr.io/myproject/my_private_repo:latest" \

Reviewer: ```suggestion
gradient experiments run ... --container "gcr.io/myproject/my_private_repo:latest" \
```

--registryUsername "_json_key" --registryPassword "$(cat keyfile.json)"
```

> Note: do not base64 encode the contents of the keyfile. It will be encoded by the gradient job runner at runtime.

Reviewer: ```suggestion
Note: do not base64 encode the contents of the keyfile. It will be encoded by the gradient experiment runner at runtime.
```
